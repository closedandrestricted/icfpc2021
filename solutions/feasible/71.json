{"vertices":[[35,51],[11,44],[30,44],[61,52],[53,24],[35,47],[57,51],[20,71],[39,74],[48,53],[25,34],[30,60],[17,103],[8,60],[19,103],[55,45],[46,42],[19,58],[25,76],[31,56],[29,75],[49,83],[39,75],[17,30],[2,81],[14,82],[2,53],[19,103],[68,56],[0,53],[10,55],[19,65],[54,53],[17,73],[33,44],[19,68],[63,11],[6,24],[39,74],[24,27],[26,49],[34,44],[29,22],[30,44],[4,43],[53,51],[29,78],[33,34],[18,48],[47,54],[15,66],[12,20],[2,49],[23,71],[28,86],[37,39],[41,48],[42,78],[39,74],[47,103],[67,69],[55,43],[54,70]]}