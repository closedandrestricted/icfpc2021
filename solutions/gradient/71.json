{"vertices":[[108,98],[83,100],[32,20],[106,72],[46,47],[108,98],[57,34],[98,124],[0,67],[69,65],[74,36],[72,10],[32,50],[78,103],[79,98],[31,56],[51,36],[81,92],[89,124],[60,55],[102,80],[62,74],[63,66],[54,83],[105,107],[91,46],[81,76],[147,71],[29,72],[81,91],[80,96],[152,136],[143,101],[142,101],[28,109],[177,89],[105,131],[110,86],[63,65],[125,163],[79,88],[139,68],[104,95],[151,81],[168,61],[90,63],[53,131],[142,134],[56,99],[121,127],[178,85],[111,91],[141,100],[140,79],[152,107],[174,86],[166,70],[186,48],[179,47],[209,52],[188,75],[212,92],[234,76]]}