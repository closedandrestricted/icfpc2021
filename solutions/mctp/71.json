{"vertices":[[14,99],[37,90],[17,77],[3,76],[40,57],[14,100],[42,91],[15,72],[15,73],[23,81],[10,54],[16,79],[51,67],[12,70],[47,74],[7,42],[2,75],[48,65],[24,90],[23,73],[38,46],[13,67],[17,74],[29,44],[20,67],[32,42],[2,52],[47,79],[42,50],[48,65],[12,40],[2,76],[4,40],[17,73],[6,42],[55,113],[8,40],[18,71],[19,75],[3,38],[40,93],[46,90],[40,23],[50,87],[76,92],[14,102],[18,73],[4,77],[5,44],[10,2],[59,112],[63,69],[33,60],[46,75],[2,38],[17,93],[0,78],[17,54],[19,59],[41,39],[36,35],[8,29],[11,56]]}