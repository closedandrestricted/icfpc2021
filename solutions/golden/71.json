{"vertices":[[110,97],[125,116],[29,20],[106,72],[46,46],[108,98],[54,33],[97,124],[0,66],[68,65],[74,36],[71,10],[32,49],[78,103],[79,98],[30,77],[51,36],[82,90],[89,124],[60,55],[102,80],[62,73],[62,66],[54,83],[105,107],[91,46],[81,76],[147,71],[29,72],[81,91],[80,96],[152,129],[141,95],[145,101],[28,109],[177,89],[112,146],[111,88],[63,65],[152,168],[79,88],[139,68],[105,111],[151,81],[168,61],[51,90],[53,131],[148,134],[58,99],[131,138],[178,85],[111,91],[141,100],[202,64],[152,107],[173,84],[166,70],[185,47],[180,47],[210,52],[190,74],[213,92],[235,76]]}